Documentation/operations/upgrade.rst

```diff
@@ -312,6 +312,8 @@ Removed Options
 ~~~~~~~~~~~~~~~
 
 * The unused flag ``sidecar-istio-proxy-image`` has been removed.
+* The flag ``endpoint-status`` has been removed.
```

Comment:
I think we still have some docs referring to the endpoint status. We should remove or update those sections:
https://docs.cilium.io/en/stable/security/aws/#validate-that-derived-policy-is-in-place (last paragraph)

Reply:
Hmm good point. I guess it didn't work before without setting `endpoint-status=policy` explicitly, which the doc doesn't mention :confused: 

Anyway, I will replace it with `cilium-dbg` command.